### PR TITLE
MEPTS-1385 | PrEP CT Diff CCS DB

### PR DESCRIPTION
**Pull request by ItsMurumba**

### CODE REVIEW

File: api/src/main/resources/Template_PEPFAR_MER_2.7_Quarterly_v1.1.xls
Comment (psmatsinhe) [inline api/src/main/resources/Template_PEPFAR_MER_2.7_Quarterly_v1.1.xls:1]: Are we still using this file? *Template*

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@ItsMurumba have a look

File: api/src/main/resources/Template_PEPFAR_MER_2.7_Quarterly_v1.1.xls
Comment (ItsMurumba) [reply api/src/main/resources/Template_PEPFAR_MER_2.7_Quarterly_v1.1.xls:1]: Yes we are

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**